pt-BR/mail/chrome/messenger/preferences/advanced.dtd

```diff
@@ -59,85 +59,85 @@
 <!ENTITY showUpdates.accesskey          "H">
 
 <!-- Networking and Disk Space -->
-<!ENTITY showConnections.label         "Connection...">
+<!ENTITY showConnections.label         "Conexão...">
 <!ENTITY showConnections.accesskey     "n">
 
 <!ENTITY offlineCompact.label "Condensar pastas se puder liberar pelo menos">
 <!ENTITY offlineCompact.accesskey "n">
 <!ENTITY kb.label "KB">
 
 <!-- Certificates -->
-<!ENTITY certSelection.label "When a server requests my certificate:">
-<!ENTITY certs.auto                      "Select one automatically">
-<!ENTITY certs.ask                       "Ask me every time">
+<!ENTITY certSelection.label "Quando um servidor solicitar meu certificado:">
+<!ENTITY certs.auto                      "Selecionar um automaticamente">
+<!ENTITY certs.ask                       "Sempre perguntar">
 
 <!ENTITY viewCertificates.label         "Certificados">
 <!ENTITY viewCertificates.accesskey     "C">
 <!ENTITY viewCRLs.label                 "Revogações">
 <!ENTITY viewCRLs.accesskey             "R">
-<!ENTITY viewOCSP.label                 "Verification">
+<!ENTITY viewOCSP.label                 "Verificação">
 <!ENTITY viewOCSP.accesskey             "f">
 <!ENTITY viewSecurityDevices.label      "Dispositivos de segurança">
 <!ENTITY viewSecurityDevices.accesskey  "D">
 
 <!-- Web Services -->
-<!ENTITY webservicesInfo.label          "Configure the Web services used within Postbox.">
+<!ENTITY webservicesInfo.label          "Configurar os Web services usados dentro do Postbox.">
 
-<!ENTITY textStartUp   "When starting up:">
-<!ENTITY radioRememberPrevState "Remember previous online state">
+<!ENTITY textStartUp   "Ao iniciar:">
+<!ENTITY radioRememberPrevState "Lembrar o status de conexão anterior">
 <!ENTITY radioRememberPrevState.accesskey "R">
-<!ENTITY radioAskMe "Ask me for online state at startup">
+<!ENTITY radioAskMe "Perguntar o status de conexão ao iniciar">
 <!ENTITY radioAskMe.accesskey "u">
-<!ENTITY radioAlwaysOnline "Always start up online">
+<!ENTITY radioAlwaysOnline "Sempre iniciar no status online">
 <!ENTITY radioAlwaysOnline.accesskey "l">
 
-<!ENTITY textGoingOnline1 "When going online:">
-<!ENTITY radioAutoSend1 "Send unsent messages">
-<!ENTITY radioNotSend1 "Do not send unsent messages">
-<!ENTITY radioAskUnsent1 "Ask to send unsent messages">
+<!ENTITY textGoingOnline1 "Ao ficar online:">
+<!ENTITY radioAutoSend1 "Enviar mensagens não-enviadas">
+<!ENTITY radioNotSend1 "Não enviar mensagens não-enviadas">
+<!ENTITY radioAskUnsent1 "Perguntar se deve-se enviar mensagens não-enviadas">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY radioAskUnsent1 "Perguntar o que fazer às mensagens não-enviadas">